fast64_internal/oot/oot_f3d_writer.py

```diff
@@ -630,6 +639,10 @@ class OOTDynamicMaterialDrawLayerProperty(bpy.types.PropertyGroup):
 	segmentB : bpy.props.BoolProperty()
 	segmentC : bpy.props.BoolProperty()
 	segmentD : bpy.props.BoolProperty()
+	customCall0 : bpy.props.BoolProperty()
+	customCall0_seg : bpy.props.StringProperty()
+	customCall1 : bpy.props.BoolProperty()
+	customCall1_seg : bpy.props.StringProperty()
```

Comment:
```suggestion
	customCall0_seg : bpy.props.StringProperty(description="Segment address of a display list to call")
	customCall1 : bpy.props.BoolProperty()
	customCall1_seg : bpy.props.StringProperty(description="Segment address of a display list to call")
```
I plan to eventually add a description for every property to make it more user friendly, may as well start now.
Those are displayed in the tooltip when hovering over a property, imo it's a great way to document whatever a property is for